### DISCUSSION TITLE
2023/05/06/mi-ctf-2023-writeup/index

### DISCUSSION THREAD

**Discussion by giscus** [sumy7/sumy7.github.io#6] [Announcements]
# 2023/05/06/mi-ctf-2023-writeup/index

SumyGG's Blog

https://sumygg.com/2023/05/06/mi-ctf-2023-writeup/index.html

**Comment by xavierskip** [comment]
我靠，这么多题！

**Reply by sumy7** [reply]
而且战线特别长，那几周每天的工作就是摸鱼、谷歌、做题。。。

**Reply by xavierskip** [reply]
那，上班打CTF还挺爽的。

**Reply by xavierskip** [reply]
出题人辛苦呀。

**Reply by sumy7** [reply]
当时看到这么多题是绝望的。。。给安全部的出题大佬们点赞。。。
